src/saml20_clj/crypto.clj

```diff
@@ -27,6 +27,13 @@
    :excl-omit-comments SignatureConstants/ALGO_ID_C14N_EXCL_OMIT_COMMENTS
    :excl-with-comments SignatureConstants/ALGO_ID_C14N_EXCL_WITH_COMMENTS})
 
+(defn has-private-key? [credential]
+  (when-let [^Credential credential (try
+                                        (coerce/->Credential credential)
+                                        (catch Throwable _
+                                          (coerce/->Credential (coerce/->PrivateKey credential))))]
```

Comment:
Can you add a test that hits this line? I'm guessing calling the function with just `test/sp-private-key` would do it right?